### PR TITLE
Fix: Remove duplicate let block to create a stock location factory

### PR DESCRIPTION
**Pull request by bsodmike**
This is a minor patch to the Shipment spec, removing a duplicate let block.

### CODE REVIEW

**Comment by jhawthorn** [comment]
:+1: Thanks Mike. Nice catch.

**Comment by cbrunsdon** [comment]
:+1:

**Comment by bsodmike** [comment]
Thanks guys - might be a good idea going through an cleaning up all the 'shipment' references to subject; I'm komotose tonight, but I'll pick it up later on in the week if no one has gotten to it by then :)

File: core/spec/models/spree/shipment_spec.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
 require 'benchmark'
 
-describe Spree::Shipment, :type => :model do
-  let(:stock_location) { create(:stock_location) }
+describe Spree::Shipment, type: :model do
+  subject { shipment }
```
Comment (athal7) [inline core/spec/models/spree/shipment_spec.rb]: are we using subject somewhere? i am not able to find where

Comment (bsodmike) [inline core/spec/models/spree/shipment_spec.rb]: I added subject but need to clean it up replacing all the shipment calls.
Will do that in the morning if someone else doesn't beat me to it :D

On Thursday, 24 December 2015, Andrew Thal <notifications@github.com> wrote:

> In core/spec/models/spree/shipment_spec.rb
> <https://github.com/solidusio/solidus/pull/613#discussion_r48372446>:
>
> > @@ -1,8 +1,8 @@
> >  require 'spec_helper'
> >  require 'benchmark'
> >
> > -describe Spree::Shipment, :type => :model do
> > -  let(:stock_location) { create(:stock_location) }
> > +describe Spree::Shipment, type: :model do
> > +  subject { shipment }
>
> are we using subject somewhere? i am not able to find where
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/solidusio/solidus/pull/613/files#r48372446>.
>


-- 
--
Mike's on the move!

Comment (athal7) [inline core/spec/models/spree/shipment_spec.rb]: for what it's worth i think keeping shipment is better because it's more explicit

Comment (bsodmike) [inline core/spec/models/spree/shipment_spec.rb]: Sure, will push that up, thanks @athal7 :)

**Pull request merged**